Dockerfile-buster-slim.template

```diff
@@ -22,12 +22,16 @@ RUN apt-get update && apt-get upgrade -y && \
 	apt-get clean && \
 	rm -rf /var/lib/apt/lists/* 
 
+RUN groupadd --gid 1000 idsvr \
```

Comment:
If the process running with uid and gid 1000, files created on a shared volume will get that as an owner. That might get unexpected consequences if the host system has a user with this uid/gid already created. IINM 1000 is the uid of the first user created on an ubuntu system, so it's likely to happen. 
I think it's hard to make the user configurable, since we're creating files. 

I suggest finding better values, and documenting it carefully. Maybe we should consider doinng this in a separate image name, since this is a breaking change as well.

Reply:
I don't think that's how it works, as far as i could test at least.
I mounted an empty volume into the container, and created a file from within the container. The file got the correct permissions inside the docker (user idsvr) and also the correct permissions in the operating system (the user i used to run docker with). Same when mounting a file that already exists in the host

Reply:
If you've verified this, then docker must have updated this behaviour, and that makes it so much easier. Great!